src/main/java/teammates/common/datatransfer/questions/FeedbackContributionQuestionDetails.java

```diff
@@ -255,12 +267,23 @@ private List<String> getTeamsWithAtLeastOneResponse(
 
     @Override
     public List<String> validateQuestionDetails() {
-        return new ArrayList<>();
+        List<String> errors = new ArrayList<>();
+
+        if (isZeroSum && isNotSureAllowed) {
+            errors.add(CONTRIB_ERROR_INVALID_OPTION);
+        }
+
+        return errors;
     }
 
     @Override
     public List<String> validateResponsesDetails(List<FeedbackResponseDetails> responses, int numRecipients) {
         List<String> errors = new ArrayList<>();
+        int actualTotal = 0;
+        boolean isAllNotSubmitted = responses
+                .stream()
+                .map(r -> ((FeedbackContributionResponseDetails) r).getAnswer() == Const.POINTS_NOT_SUBMITTED)
+                .reduce(true, (acc, curr) -> acc && curr);
```

Comment:
- Consider:
  ```patch
  +                .allMatch(r -> ((FeedbackContributionResponseDetails) r).getAnswer() == Const.POINTS_NOT_SUBMITTED);
  -                .map(r -> ((FeedbackContributionResponseDetails) r).getAnswer() == Const.POINTS_NOT_SUBMITTED)
  -                .reduce(true, (acc, curr) -> acc && curr);
  ```
- You may also want to consider early return if `isAllNotSubmitted` is true, since there is nothing to validate if there is no responses.